common/src/androidMain/kotlin/com/cuhacking/atlas/common/DataCache.kt

```diff
@@ -0,0 +1,27 @@
+package com.cuhacking.atlas.common
+
+import android.content.Context
+import kotlinx.coroutines.withContext
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import java.io.File
+
+actual class DataCache {
+    lateinit var appContext: Context
+    actual var lastModified: Instant? = null
```

Comment:
```suggestion
    actual var lastModified: Instant? = null
        private set
```
We don't want the `lastModified` time to be modified from outside.